examples/vue-jsx/vite.config.ts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from 'vite'
+import Vue from '@vitejs/plugin-vue'
+import Jsx from '@vitejs/plugin-vue-jsx'
+
+export default defineConfig({
+  plugins: [Vue(), Jsx()],
+  test: {
+    global: true,
+    environment: 'happy-dom',
+    ssrTransformInclude: [/.[tj]sx$/],

```

Comment:
Have one side include isn't enough, we could have:

```ts
{
  transformMode: {
    ssr: [...], // directly ssr transform
    client: [...], // client transform + ssr transform
  }
}
```

Reply:
I am a little bit confused with terminology here

ssr is this? https://github.com/vitest-dev/vitest/pull/469/files#diff-cdd740fae25ab6905acf643dbfb4336f419a6fdfe4a6c73e61868063e1cc5abeR25

and client is this? https://github.com/vitest-dev/vitest/pull/469/files#diff-cdd740fae25ab6905acf643dbfb4336f419a6fdfe4a6c73e61868063e1cc5abeL28-L32

Reply:
Also one contradicts another, so do wee really need this? Or you have something else in mind concerning how we transform files?

Reply:
Or you mean something like this?

```ts
if(
  (id.match(...) /* current matcher */ || isIncludes(config.transformMode.client, id)) 
  && !isIncludes(config.transformMode.ssr, id)
) {
  transformRequest()
} else {
  transformRequest()
  ssrTransform()
}
```

Reply:
Let me do it

Reply:
I am letting you do it! But I would still like an explanation 👀